kong/clustering/services/sync/hooks.lua

```diff
@@ -161,8 +166,38 @@ function _M:register_dao_hooks()
 
     ngx_log(ngx_DEBUG, "[kong.sync.v2] new delta due to deleting ", name)
 
-    -- set lmdb value to ngx_null then return row
-    return self:entity_delta_writer(entity, name, options, ws_id, true)
```

Comment:
It seems that we lost this logic in the new patch? Is it intended?

Reply:
Yes, `entity_delta_writer()` only process one entity, but in deletion we may have many entities, but the basic logic is same, please compare to `entity_delta_writer()`.